src/ast/seq_decl_plugin.h

```diff
@@ -478,10 +493,11 @@ class seq_util {
         bool is_loop(expr const* n, expr*& body, unsigned& lo) const;
         bool is_loop(expr const* n, expr*& body, expr*& lo, expr*& hi) const;
         bool is_loop(expr const* n, expr*& body, expr*& lo) const;
-        unsigned min_length(expr* r) const;
+        unsigned min_length(expr* r);
```

Comment:
declare m_info and m_pinned with "mutable", then we can keep the const qualifier, which is probably reasonable.